File: Faltantes o por revisar/TestCasesRandomStrings.cpp

#include<bits/stdc++.h> 
using namespace std;

#define RUN 5
#define MAX 25 
#define MAXLEN 100 

void randString() {
    srand(time(NULL));
    int LEN; 
    for (int i = 1; i <= RUN; i++) {
        LEN = 1 + rand() % MAXLEN;
        printf("%d\n", LEN);
        for (int j = 1; j <= LEN; j++)
            printf("%c", 'a' + rand() % MAX);
        printf("\n");
    }
}

int main() {
    randString();
}

File: Faltantes o por revisar/TestCasesRandomStrings.py
import random as r
RUN=5
MAX=25
MAXLEN=100
def randString():
    for i in range(RUN):
        LEN=1+r.randint(0,MAXLEN)
        print(LEN)
        STR=""
        for j in range(LEN):
            STR+=chr(ord('a') + r.randint(0,MAX))
        print(STR)
        
randString()